### DISCUSSION TITLE
권지은 - 엔터테인먼트 - 넷플릭스 컨텐츠 데이터 분석

### DISCUSSION THREAD

**Discussion by ndyakwon** [dinnercoding/wanted-pre-onboarding-challenge-data#99] [2023년 11월 프리온보딩 데이터 챌린지]
## 도메인 선정 이유
- 최근 간단한 추천 알고리즘 개발을 목표로 공부 중입니다. 유용한 인사이트를 얻고자 넷플릭스 데이터를 선정했습니다.
## 구체적인 문제 정의
- 넷플릭스 컨텐츠 현황을 파악하여 추천 알고리즘 개발을 위한 인사이트를 얻고자 합니다.
- '국가에 따라 주로 생산되는 컨텐츠의 속성이 다를 것'이라는 가설 하에, 한국 넷플릭스 컨텐츠의 특수성을 분석하고자 합니다.
## 데이터 업로드
- Netflix Movies and TV Shows Dataset (출처 : Kaggle)

**Comment by ghost** [comment]
첫번째로 제출해주셨군요!! 감사합니다 지은님 :)
문제 정의도 구체적으로 잘 정의해주셨네요. 본인만의 가설을 잘 세우셨고 그 과정을 데이터에 기반하여 분석해나가시는 과정이 기대가 됩니다. ㅎㅎ 데이터를 분석하다 보면 처음에 세웠던 가설이 맞을 수도 있지만 완전히 뒤바뀔수도 있고 새로운 가설을 세우실 수도 있을 텐데요~ 어떠한 흐름으로 사고 과정이 변해가는지 잘 기록해두시면 좋은 경험이 되실거라 생각합니다!
